osf/models/metaschema.py

```diff
@@ -11,7 +11,7 @@
 from website.project.metadata.utils import create_jsonschema_from_metaschema
 
 
-class MetaSchema(ObjectIDMixin, BaseModel):
```

Comment:
"MetaSchema" usages should also be removed from the frontend, e.g. https://github.com/CenterForOpenScience/osf.io/blob/df4d8832bbee66e744d10a11e7c568c9c4f57fd3/website/templates/project/project_base.mako#L212

Reply:
@sloria, @erinspace asked me about this and my opinion was to leave the frontend as-is since it's legacy and to reduce the number of lines affected. I don't feel super strongly about it, but that's my two cents.

Reply:
Fine with me either way.